sql/modules/Menu.sql

```diff
@@ -12,13 +12,14 @@ CREATE TYPE menu_item AS (
    label varchar,
    path varchar,
    parent int,
-   args text[]
+   args text[],
```

Comment:
Changing a type doesn't work on existing databases. You'll need to `DROP CASCADE` it first (`IF EXISTS`, of course because otherwise it doesn't work on new databases).